test/runtests.jl

```diff
@@ -91,6 +91,8 @@ import FillArrays: AbstractFill, RectDiagonal, SquareEye
                 Fill{Float32}(one(Float32),5,5)
 
         @test Fill{T}(F) ≡ Fill{T,2}(F) ≡ typeof(F)(F) ≡ F
+
+        show(devnull, MIME("text/plain"), F)
```

Comment:
What is this call for?

Reply:
This call is to provide code coverage for the new `show` method.
Without some call like this, the code coverage for the package will be reduced.